packages/angular/projects/ui-angular/src/lib/primitives/button/button.component.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { ButtonComponent } from './button.component';
+import { TestBed, ComponentFixture } from '@angular/core/testing';
+
+describe('ButtonComponent', () => {
+  let fixture: ComponentFixture<ButtonComponent>;
+  let component: ButtonComponent;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ButtonComponent],
+    }).compileComponents();
```

Comment:
Out of curiosity, why do we need to use `TestBed` in for the `ButtonComponent` test but not for `CheckboxComponent`?

Reply:
I wanted to show in our tests some tests run with the Angular testbed and some without (the button and checkbox were written by @SantoshLatpate then I updated the button).  The testbed is is great for more complicated tests, with interactions with services and DOM. While testing classes alone are great for tests that are more simpler or don't require the DOM. 

The official Angular [documentation](https://angular.io/guide/testing-components-basics) leans more heavily on testbed, but explains you can use both. I feel like we should show both ways of doing it.

Reply:
Cool, thanks for the explanation 👍🏼